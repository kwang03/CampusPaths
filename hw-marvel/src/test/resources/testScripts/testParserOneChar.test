#Tests the parser in creating a graph with one character

#Create Graph
LoadGraph myGraph oneChar.csv

ListNodes myGraph

ListChildren myGraph Char1