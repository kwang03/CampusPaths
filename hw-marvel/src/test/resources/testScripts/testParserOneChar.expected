#Tests the parser in creating a graph with one character

#Create Graph
loaded graph myGraph

myGraph contains: Char1

the children of Char1 in myGraph are: